### PR TITLE
Cleanup user guide sections a-c

### PR DESCRIPTION
**Pull request by kprosise**
Account management, container preloading, custom CI, and custom SOTA client sections were all cleaned up in line with the style guide. Container preloading was also updated to remove references of named branches (devel), and removed a number of screenshots of the UI which are covered in the getting started and tutorial sections.

QA Steps: viewed rendered html in browser. Ran vale linter and linkcheck.

This commit applies to FFTK-988

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Just some basic cleanup and minor updating.

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

If any major changes need to be made to the content to reflect recent changes, the can be handled as a separate commit; the focus here is largely on style.

### CODE REVIEW

**Comment by kprosise** [comment]
I will fix that build error shortly!

**Review by angolini — APPROVED** [review]
NICE!!!! Thanks a lot!

File: source/user-guide/container-preloading/container-preloading.rst [hunk 1]
```diff
@@ -161,24 +158,21 @@ A user can try to start preloaded Restorable Apps manually by using `aklite-apps
 Starting Compose Apps Automatically
 -----------------------------------
 
-To start the preloaded application automatically after the boot and before
-the device registration when aktualizr-lite starts, you have to enable a systemd service
-responsible for it.
+To start the preloaded application automatically between the boot and the device registration when aktualizr-lite starts,
+you have to enable a systemd service responsible for it.
```
Comment (kprosise) [inline source/user-guide/container-preloading/container-preloading.rst:162]:
```suggestion
enable a systemd service responsible for it.
```

**Comment by doanac** [comment]
Docs for 1a09e8229ce76275168f459957db70b8a547773c are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2317/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 8d4cf8f31f8648ac01276267a80cc402b2032a80 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2318/docs/artifacts/html/index.html

**Pull request merged**